src/cforth/util.fth

```diff
@@ -9,14 +10,8 @@
    then
 ; immediate
 
-: s(  
-   [char] ) parse ;
-
-: .(
-  s( type ;
-
 : ok ;
-: trigger  0 0 2>r 2r>  drop drop  ; immediate
+: trigger  0 0 2>r 2r>  drop drop  ; immediate             \ what's that for? "magic" 0. return stack marker?
```

Comment:
This was for triggering a logic analyzer in some scenario.  I forget the details.  It could go away.